MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -50,7 +58,8 @@ public sealed class CreditCard
             string? bankPhoneNumber = null,
             char? avsResult = null,
             char? cvvResult = null,
-            string? token = null
+            string? token = null,
+            bool? was3dSecureSuccessful = null
```

Comment:
This will break binary compatibility as optional arguments are determined at compile time. To fix this, add another constructor with no optional arguments that otherwise matches the old constructor.